liblangutil/Exceptions.h

```diff
@@ -189,6 +191,19 @@ class Error: virtual public util::Exception
 		return "Error";
 	}
 
+    static std::optional<Severity> stringToSeverity(std::string _severity)
```

Comment:
I think this name would better match how we name things in various places:
```suggestion
    static std::optional<Severity> severityFromString(std::string _severity)
```

Reply:
Done 👍